.github/workflows/scripts/check_docstrings.py

```
@@ -0,0 +1,773 @@
+            if bool(docstring.fatal) is False:
+                bad = match_arguments_to_docstring(
+                    function, docstring, line_number
+                )
+                if bool(bad):
+                    violations.extend(bad)
+
+    # Return
+    return violations
+
+
+def ignore_function(function, file_path):
```

Ruff (0.11.9)

Missing return type annotation for public function `ignore_function`

(ANN201)